xml/core_registers.xml

```diff
@@ -90,8 +90,12 @@
     </register>
 
     <register name="Debug PC" short="dpc" address="0x7b1">
-        Upon entry to debug mode, \Rdpc is written with the
-        virtual address of the instruction that encountered the exception.
+        Upon entry to debug mode, \Rdpc is updated with the virtual address of
+        the next instruction to be executed.  {\tt ebreak} instructions are an
```

Comment:
I think this is less clear. "The instruction that encountered the exception" is what it is. It's only relevant that it is "the next instruction to be executed" when you are EXITING debug mode.

I think the confusion is that entry to debug mode is not always caused by an exception. "Halt Request" is not really defined as an exception. It acts like an interrupt. We should just use the same language as the Priv spec uses for explaining how to update MEPC when there is an interrupt.